charts/registratie-component/templates/configmap-nginx.yaml

```
@@ -0,0 +1,74 @@
+{{- if not .Values.nginx.existingConfigmap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "registratie-component.nginxFullname" . }}
+  labels:
+    {{- include "registratie-component.nginxLabels" . | nindent 4 }}
+data:
+  proxy: |
+    proxy_pass_header Server;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_connect_timeout 300s;
+    proxy_read_timeout 300s;
+    proxy_redirect off;
+    proxy_pass_request_headers on;
+    proxy_http_version 1.1;
+    proxy_pass http://{{ include "registratie-component.fullname" . }}.{{ .Release.Namespace }}:{{ .Values.service.port }};
+    {{- if .Values.settings.useXForwardedHost }}
+    proxy_set_header X-Forwarded-Host $http_host;
+    {{ else }}
+    proxy_set_header Host $http_host;
+    {{- end }}
+
+  default.conf: |
+    server {
+        listen 8080 default_server;
+        server_name {{ .Values.settings.allowedHosts  | replace "," " "}} localhost;
+        server_tokens off;
+
+        gzip  on;
+        gzip_http_version 1.0;
+        gzip_comp_level 2;
+        gzip_min_length 1100;
+        gzip_buffers     4 8k;
+        gzip_proxied any;
+        gzip_types
+
+          # text/html is always compressed by HttpGzipModule
+          text/css
+          text/javascript
+          text/xml
+          text/plain
+          text/x-component
+          application/javascript
+          application/json
+          application/xml
+          application/rss+xml
+          font/truetype
+          font/opentype
+          application/vnd.ms-fontobject
+          image/svg+xml;
+
+        gzip_static on;
+
+        gzip_proxied        expired no-cache no-store private auth;
+        gzip_disable        "MSIE [1-6]\.";
+        gzip_vary           on;
+
+        location / {
+            include conf.d/proxy;
```

Hi,

Zoals besproken met Sergei, we moeten deze toevoegen, zelfde als openzaak:

```suggestion
            include conf.d/proxy;
          location /documenten/api/v1/enkelvoudiginformatieobjecten {
              client_max_body_size 4G;
              include conf.d/proxy;            
            
```

https://github.com/maykinmedia/charts/blob/b73e9dc8a2e986af505fdeb7491132024f7eb067/charts/openzaak/templates/configmap-nginx.yaml#L67C21-L67C31

---

Maar de registratie component heeft geen `/documenten/api/v1/enkelvoudiginformatieobjecten` endpoint? of wel?

---

Klopt, van @sergei-maertens had ik begrepen dit endpoint. `/api/v1/documenten/:uuid/bestandsdelen/:uuid`. En dat deze net zo werkt als openzaak en we dus max_body_size op dit endpoint ook naar 4G willen? @sergei-maertens corrigeer me als ik dit verkeerd heb begrepen.

---

klopt! De endpoint is deze: https://woo-publicaties.test.opengem.nl/api/v1/docs/#tag/Documenten/operation/documentenBestandsdelenUpdate

Die proxiet de uploads naar de acherliggende documenten API, dus de instelling voor `client_max_body_size` moet daarop afstembaar zijn. Ik denk dat dit dus een variabele moet zijn - als je een documenten API hebt die 8GB chunks toelaat, dan moeten wij ook 8GB toelaten.

---

Ik heb het aangepast!